app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -142,7 +142,15 @@ authselect --useshadow --passalgo=<%= @host.operatingsystem.password_hash.downca
 <% else -%>
 authconfig --useshadow --passalgo=<%= @host.operatingsystem.password_hash.downcase || 'sha256' %> --kickstart
 <% end -%>
-timezone --utc <%= host_param('time-zone') || 'UTC' %> <%= host_param('ntp-server') ? "--ntpservers #{host_param('ntp-server')}" : '' %>
+
+timezone --utc <%= host_param('time-zone') || 'UTC' %>
+<% if host_param('ntp-server') %>
+<% if rhel_compatible && os_major > 8 %>
```

Comment:
I always prefer `>=` over `>` so I don't need to do mental math. Though in this particular case you can also revert it to `rhel_compatible && os_major < 9` and swap the branches. Then you gain non-EL support like Fedora.

Reply:
`timezone --ntpservers` is also deprecated in Oracle Linux 9 and Fedora 34. Do I need to check here all supported OS?